### PR TITLE
Add new company - Replit

### PR DESCRIPTION
**Pull request by archaengel**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/replit.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+# Reaction Commerce
```
Comment (KieranRobson) [inline company-profiles/replit.md:2]: This should be `Replit`

File: README.md [hunk 2]
```diff
@@ -556,6 +556,7 @@ Name | Website | Region
 [Remote Garage](/company-profiles/remote-garage.md) | http://www.remotegarage.club/ | India
 [RemoteBase](/company-profiles/remotebase.md) | https://remotebase.com | Worldwide
 [RenoFi](/company-profiles/renofi.md) | https://renofi.com/ |  Worldwide
+[Replit](/company-profiles/replit.md) | https://replit.com/site/careers | Worldwide
```
Comment (KieranRobson) [inline README.md:559]: This link should be the main site link, not the careers page

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Two changes to be made

File: README.md (hunk 2 above)
Comment (archaengel) [reply README.md]: Done

File: company-profiles/replit.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (KieranRobson) [inline company-profiles/replit.md:1]: This line could be removed

**Review by KieranRobson — APPROVED** [review]
Although my comment is just a suggestion, this looks good to me!

File: company-profiles/replit.md (hunk 3 above)
Comment (archaengel) [reply company-profiles/replit.md]: Sgtm to keep it consistent with the other profiles in the repo. Done

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Hey there, thanks for contributing, it is appreciated

**Pull request merged**